kingpin/actors/aws/base.py

```diff
@@ -119,7 +119,8 @@ def __init__(self, *args, **kwargs):
         boto_config = botocore_config.Config(
             region_name=self.region,
             retries={
-                "mode": "adaptive",
+                "max_attempts": aws_settings.AWS_MAX_ATTEMPTS,
+                "mode": aws_settings.AWS_RETRY_MODE,
```

Comment:
It seems like we are reimplementing Boto’s environment variable usage. Would it be cleaner if we removed kingpin from that entirely by removing any explicit override?
We could document the behavior.

Reply:
I was attempting to maintain some "optimized sane defaults" in the case where the user does not specify anything. Because kingpin can be high call volume, the defaults for boto3 may not be suitable OOB.